Formula/chakra.rb

```diff
@@ -0,0 +1,28 @@
+class Chakra < Formula
+  desc "The core part of the Chakra Javascript engine that powers Microsoft Edge"
+  homepage "https://github.com/Microsoft/ChakraCore"
+  url "https://github.com/Microsoft/ChakraCore/archive/v1.4.0.tar.gz"
+  sha256 "f6b4a3bce9e69bb3ea619a82805460906ef53d4f9da98b51e554dd48d6cdfdb3"
+
+  depends_on "cmake" => :build
+  depends_on "icu4c" => :optional
+
+  def install
+    if build.with? "icu4c"
+      icu4c = Formula["icu4c"]
+      icu = "--icu=#{icu4c.opt_include}"
+    else
+      icu = "--no-icu"
+    end
+
+    system "./build.sh", "--static", "--test-build", icu
```

Comment:
The test build enables command-line flags so most of the time people will want a test build. I’ll make this an option then